includes/optional-modules/class-rss.php

```diff
@@ -505,6 +539,34 @@ public static function modify_feed_query( $query ) {
 		if ( ! empty( $settings['category_exclude'] ) ) {
 			$query->set( 'category__not_in', array_map( 'absint', $settings['category_exclude'] ) );
 		}
+
+		if ( ! empty( $settings['update_frequency'] ) ) {
+			// Split the string on the hyphen to get the update frequency and the number of times to update.
+			$settings['update_frequency'] = explode( '-', $settings['update_frequency'] );
+			add_filter(
+				'rss_update_period',
+				function() use ( $settings ) {
+					return $settings['update_frequency'][0];
+				}
+			);
+			add_filter(
+				'rss_update_frequency',
+				function() use ( $settings ) {
+					return $settings['update_frequency'][1];
```

Comment:
For the case of 'daily', explode will return an array with only one item.

Reply:
Good catch! Fixed in 25156bb3406b34849e46ee39aa0db003291edfed